env0/resource_template.go

```diff
@@ -199,6 +199,13 @@ func resourceTemplateRead(ctx context.Context, d *schema.ResourceData, meta inte
 	d.Set("revision", template.Revision)
 	d.Set("type", template.Type)
 	d.Set("terraform_version", template.TerraformVersion)
+
+	var rawSshKeys []map[string]string
+	for _, sshKey := range template.SshKeys {
+		rawSshKeys = append(rawSshKeys, map[string]string{"id": sshKey.Id, "name": sshKey.Name})
```

Comment:
why we need this loop?
looks like `template.SshKeys` already contains what we need no?

Reply:
That's the whole fix basically.
Terraform schema expects a "raw" type (as it's defined with `TypeMap` for this attribute) so we must convert it from a custom type such as `TemplateSshKey` to the expected `map[string]string`

Reply:
so can't we change it from "raw" map to out TempalteSshKey?

Reply:
Nope, schema doesn't support non raw types.